daemon/openrazer_daemon/dbus_services/dbus_methods/all.py

```diff
@@ -203,3 +203,86 @@ def get_matrix_dims(self):
     self.logger.debug("DBus call has_matrix")
 
     return list(self.MATRIX_DIMS)
+
+
+@endpoint('razer.device.misc', 'setFanSpeed', in_sig='yy')
+def set_fan_speed(self, fan_id, fan_speed):
+    """
+     set fan speed
+     :param fan_id: might change fans separately if more than one available (ususally 0x00)
+     :param fan_speed: speed in RPM/100
+     Values appear to be RPM/100, observed were 0x2b - 0x36, i.e. 4300RPM - 5400RPM
+    """
+
+    self.logger.debug("DBus call set_fan_speed")
+
+    driver_path = self.get_driver_path('fan_{0:d}_speed'.format(int(fan_id)))
+
+    payload = bytes([fan_speed])
+
+    with open(driver_path, 'wb') as driver_file:
+        driver_file.write(payload)
+
+
+@endpoint('razer.device.misc', 'getFanSpeed', in_sig='y', out_sig='y')
+def get_fan_speed(self, fan_id):
+    """
+     get fan speed
+     :return fan_speed: speed in RPM/100
+     Values appear to be RPM/100, observed were 0x2b - 0x36, i.e. 4300RPM - 5400RPM
```

Comment:
Dealing with bytes for scripts/apps seems too low level for higher level API usage. May we have the daemon convert them to an `int` to get/set a RPM value? 

Plus, may we add a variable for the min/max RPM values? Hardcoding the value should be fine, until it's known a laptop could spin faster then 5400 RPM, or less then 4300 RPM.